app/poros/movie_call.rb

```diff
@@ -13,8 +13,8 @@ def initialize(data)
    @api_id = data[:id]
    @title = data[:title]
    @summary = data[:overview]
-   @hours = (data[:runtime] / 60)
```

Comment:
I think we may need to keep this - I have a version on my end that might work - lets merge this for now and see how it does